bin/lib/package-json.js

```diff
@@ -21,5 +22,14 @@ export function addScriptToPackageJson(scriptName, scriptCommand, { appDir = pro
     packageJson.scripts = {};
   }
   packageJson.scripts[scriptName] = scriptCommand;
+
+  if (!packageJson.dependencies) {
+    packageJson.dependencies = {};
+  }
+  if (!packageJson.dependencies['cross-env']) {
+    packageJson.dependencies['cross-env'] = `^${execSync('npm show cross-env version')
+      .toString()
+      .trim()}`;
+  }
```

Comment:
Wouldn't we need to run `npm/yarn` install after this step?

Is it impossible for us to depend on `cross-env` and still use it somehow in a script?

Reply:
We're adding a script to the user's package.json. A script that uses cross-env.
So we should also add that dependency to their package.json.

I don't think we should attempt to actually install the package.

Reply:
But I mean its not going to work when they try and run chromatic in that case.

Reply:
ok, it wasn't so hard to add after all.